UIKitMovie/UIKitMovie/View/ListMovieTableViewCell.swift

```diff
@@ -52,6 +68,20 @@ final class MovieTableViewCell: UITableViewCell {
         createMovieOverviewLabelConstraint()
     }
 
+    private func fetchImage(url: String, listMovieViewModel: ListMovieViewModelProtocol) {
+        listMovieViewModel.fetchImage(url: url) { [weak self] result in
```

Comment:
```suggestion
        listMovieViewModel.fetchImage(url: url) { [weak self] result in
```
это в vm сюда верни только дату